lib/src/internet_check_option.dart

```diff
@@ -50,11 +57,18 @@ class InternetCheckOption {
   /// Defaults to 3 seconds.
   final Duration timeout;
 
+  /// A map of additional headers to send with the request.
+  final Map<String, String> headers;
+
+  /// A callback to check expected response
+  final ResponseStatusFn? responseStatusFn;
```

Comment:
Can you elaborate the documentation here more.

e.g. A custom callback function to decide whether the request succeeded or not.

Something like this 👆🏼

Reply:
docs added